### PR TITLE
Exhibitors filter by name

### PR DESCRIPTION
**Pull request by beluamat29**

### CODE REVIEW

File: app/models/exhibitor.rb [hunk 1]
```diff
@@ -15,8 +15,8 @@ class Exhibitor < ApplicationRecord
   alias_attribute :video_1, :video_1__c
   alias_attribute :video_2, :video_2__c
   alias_attribute :video_3, :video_3__c
-  alias_attribute :logo, :logo_url__c
-  #alias_attribute :logo, :photourl
+  #alias_attribute :logo, :logo_url__c
+  alias_attribute :logo, :photourl
```
Comment (matiasmelendi) [inline app/models/exhibitor.rb]: :warning:

Comment (beluamat29) [reply app/models/exhibitor.rb]: :+1:

File: app/models/exhibitor.rb [hunk 2]
```diff
@@ -57,10 +57,19 @@ def logo_url
         'https://res.cloudinary.com/dnvnnka50/image/upload/v1557506866/not_available_xdxj6u.jpg'
   end
 
+  def self.search(query_string, conference)
+    search_param = "%#{query_string}%"
+
+    exhibitors = Exhibitor.
+        by_conference(conference).
+        where("account.name ILIKE ?", search_param)
```
Comment (matiasmelendi) [inline app/models/exhibitor.rb]: I'd use `Exhibitor.table_name` so the query should look like `where("#{Exhibitor.table_name}.name ILIKE ?", search_param)`

Comment (beluamat29) [reply app/models/exhibitor.rb]: :ok_hand:

File: app/models/exhibitor.rb (hunk 1 above)
Comment (matiasmelendi) [reply app/models/exhibitor.rb]: :warning: :warning:

Comment (beluamat29) [reply app/models/exhibitor.rb]: :man_facepalming: done

**Pull request merged**